src/screens/SignUp/indext.test.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { waitFor } from '@testing-library/react';
+import { mount } from 'enzyme';
```

Comment:
Yeap!, I undestand your point of view. 
I ended up doing it this way only because I had a lot of time trying to understand the testing and eventually found a way to do it using `enzyme`.
I knew beforehand that, it is not very consistent testing in terms of getting multiple libraries. But, it is very useful for my training stage, because I have seen `enzyme` is used a lot and also I worked with the `react-testing-library`. 
Thank you for ignoring this type of testing, as I probably won't handle it this way in other ones.